interpretation/deepseismic_interpretation/azureml_pipelines/README.md

```diff
@@ -0,0 +1,175 @@
+# Integrating with AzureML
```

Comment:
Could you please also add some verbiage in the main README and point to this file? AML team would love this!